### ISSUE TITLE
Setting data-content muddles up the title for the selected option

### ISSUE THREAD

**Issue by yazinsai** [silviomoreto/bootstrap-select#469]
I've added some HTML in the `data-content` attribute of the `<option>`s so it displays a flag next to the list of countries.

Problem is, when I hover over the selected option, I see a tooltip with the HTML that I entered (and it looks gross). I noticed this happens because the `title` for the displayed element takes the value of the `data-content` attribute for some reason.
![yngxe jpg](https://f.cloud.github.com/assets/1846034/2038120/3d1881bc-8985-11e3-933f-b4d0b048cad3.png)

===
Here's the detailed HTML I'm using:

```html
<option value='BH' data-content="<img src='
    data:image/png;base64,iVB....'> Bahrain"> Bahrain </option>
```

**Comment by jgambarios** [comment]
With the latest 1.6.3 the issue is present again, this was broken in this latest release, 1.6.2 works as expected.

**Comment by yazinsai** [comment]
See @jgambarios's note above .. seems like the issue has resurfaced.

**Comment by jgambarios** [comment]
Any news on this one?, would be great to have it fix for the next release. Thanks!

**Comment by Khrysller** [comment]
+1

**Comment by emage** [comment]
+!

**Comment by jgambarios** [comment]
Just in my opinion this issue is critical because all we use the **data-content** wont be able to upgrade, this is an error that I can not allow the visitors of my site to see.

**Comment by snipe** [comment]
+1

**Comment by snipe** [comment]
@jgambarios - if you'r hosting the file locally, you can apply the change in https://github.com/silviomoreto/bootstrap-select/commit/4e615930d111e94bcbfcc735c6886ba1b1621261 and it should work. (Sucks to update out of band like that, but in a pinch it might help.)

**Comment by caseyjhol** [comment]
I've been trying to replicate this on my end. It'd be nice to get this squashed finally. Can anybody link me to an example?

**Comment by snipe** [comment]
Sure thing: http://snipe.github.io/bootstrap-select-demo/

When you  mouseover the dropdown (without clicking on it), you can see the title attribute pops up this:

![screen shot 2015-04-30 at 3 55 06 pm](https://cloud.githubusercontent.com/assets/197404/7424033/628bcdfe-ef51-11e4-8e28-1b3889776fc9.png)

Applying the fix from https://github.com/silviomoreto/bootstrap-select/commit/4e615930d111e94bcbfcc735c6886ba1b1621261 (which looks like a regression issue, since it's not in the current version) solves it.

**Comment by snipe** [comment]
For easy reference, the select code is here:

	<select class="selectpicker" name="listing_type[]" title='want or have' data-size="2" data-header="Want/Have" multiple>
		<option value="want" data-content="<span class='label label-info'>have</span>" selected>have</option>
		<option value="have" data-content="<span class='label label-default'>want</span>" selected>want</option>
	</select>

With this JS at the bottom:

	<script>
	  	// To style only <select>s with the selectpicker class
	  	$('.selectpicker').selectpicker();
	</script>

**Comment by caseyjhol** [comment]
Can you update that to 1.6.3 to see if it persists? It's working for me with 1.6.5 in my test environment.

**Comment by snipe** [comment]
It's using 1.6.3 - did you mean could I update to 1.6.5?

	<link rel="stylesheet" href="http://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.6.3/css/bootstrap-select.min.css">

**Comment by snipe** [comment]
I can confirm this is fixed in 1.6.5: http://snipe.github.io/bootstrap-select-demo/

**Comment by caseyjhol** [comment]
Yes, I meant 1.6.5 - sorry. Happy to hear it's working. I'll wait for confirmation from others to ensure it's working as expected for them before closing this out.
